superset/assets/src/explore/visTypes.jsx

```diff
@@ -30,7 +30,7 @@ export const sections = {
     label: t('Publish & Subscribe'),
     expanded: true,
     controlSetRows: [
-      ['publish_columns', 'linked_slice']
+      ['publish_columns']
     ],
```

Comment:
can we retain it for backward compitability and make it hidden

Reply:
`visTypes.jsx` this file is of no more use